package.json

```diff
@@ -1,106 +1,78 @@
 {
-  "name": "SyncLounge",
-  "version": "2.0.0",
+  "name": "synclounge",
+  "version": "0.0.0-semantically-released",
   "description": "Synced playback of Plex content",
   "author": "samcm",
-  "private": false,
+  "bin": "./server.js",
+  "files": [
+    "dist",
+    "config",
+    "server.js"
+  ],
   "scripts": {
-    "dev": "node build/dev-server.js",
-    "start": "npm run build; node webapp.js",
-    "server": "node server.js",
-    "build": "node build/build.js"
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --modern",
+    "lint": "vue-cli-service lint",
+    "lint:style": "vue-cli-service lint:style",
+    "prepare": "if-env SKIP_BUILD=true && echo 'Skipping build' || npm run build"
   },
   "dependencies": {
-    "args-parser": "^1.1.0",
-    "axios": "^0.18.1",
-    "babel-eslint": "^8.2.2",
-    "body-parser": "^1.19.0",
-    "cors": "^2.8.3",
-    "express": "^4.16.3",
-    "fscreen": "^1.0.2",
-    "git-rev-sync": "^1.12.0",
-    "jsonfile": "^4.0.0",
-    "moment": "^2.18.1",
-    "sails-disk": "^1.1.0",
-    "socket.io": "^2.0.3",
-    "string-similarity": "^1.2.0",
-    "videojs-contrib-hls": "^5.15.0",
-    "vue": "^2.6.10",
-    "vue-cookies": "^1.7.0",
-    "vue-router": "^2.3.1",
-    "vuetify": "^1.5.14",
-    "vuex": "^2.3.1",
-    "waterline": "^0.13.6",
-    "waterline-mysql": "^0.6.0"
+    "nconf": "^0.10.0",
+    "syncloungesocket": "^5.3.4"
```

Comment:
Need to take a look at syncloungesocket

Reply:
It can be found https://github.com/ttshivers/syncloungesocket